tsconfig.test.json

```diff
@@ -1,13 +1,6 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "noImplicitAny": false,
-    "target": "es6",
-    "noImplicitThis": false,
-    "alwaysStrict": false,
-    "strictBindCallApply": false,
-    "strictNullChecks": false,
-    "strictFunctionTypes": false,
-    "strictPropertyInitialization": false
```

Comment:
The tests compile just fine without all these flags disabled, so I've simplified the config by aligning it as much as possible with the root config.